workflow/test_coverage.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cmake -DCMAKE_BUILD_TYPE=Coverage .
+# sh realm-sync/build.sh config -DREALM_SYNC_PREFIX=/source/realm-sync/
+make VERBOSE=1 -j2 generate-coverage-cobertura
```

Comment:
Why `-j2`? Using a value closer to the number of processors gives a significantly faster build on my machine.